pmm-app/src/pmm-qan/panel/qan.constants.ts

```diff
@@ -885,7 +880,7 @@ export const METRIC_CATALOGUE = {
   },
   tmp_table_on_disk: {
     humanizeName: 'Tmp Table on Disk',
-    tooltipText: 'The queries temporary table was stored on disk',
+    tooltipText: 'The query\'s temporary table was stored on disk',
```

Comment:
@lunaticusgreen Is this 'the table of a query' or 'all of the queries tables'?

Reply:
"Whether the query needed temporary tables on disk."  this is an alternative description of the metric (tmp_table_on_disk) I can see

Reply:
The first interpretation fits. These 'boolean' metrics are all phrased as predicates (something IS) rather than answers to a question (Whether ... ).